plugins/speculation-rules/settings.php

```diff
@@ -43,9 +43,9 @@ function plsr_get_eagerness_labels(): array {
 /**
  * Returns the default setting value for Speculative Loading configuration.
  *
- * @since 1.0.0
+ * @since n.e.x.t
```

Comment:
Oops. This should not have been changed.

```suggestion
 * @since 1.0.0
```